beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -408,21 +408,20 @@ where
         self
     }
 
-    pub fn execution_layer_from_urls(mut self, urls: &[&str]) -> Self {
+    pub fn execution_layer_from_url(mut self, url: &str) -> Self {
         assert!(
             self.execution_layer.is_none(),
             "execution layer already defined"
         );
 
-        let urls: Vec<SensitiveUrl> = urls
-            .iter()
-            .map(|s| SensitiveUrl::parse(s))
-            .collect::<Result<_, _>>()
-            .unwrap();
+        let url = match SensitiveUrl::parse(url) {
+            Ok(s) => Some(s),
+            Err(_) => None,
+        };
```

Comment:
you could add a bit of sugar here:
```suggestion
        let url = SensitiveUrl::parse(url).ok();
```